app/src/main/java/com/example/drawingactivity/DrawButtonView.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.drawingactivity
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color.*
+import android.graphics.Paint
+import android.graphics.Rect
+import android.util.AttributeSet
+import android.view.View
+import androidx.core.content.ContextCompat
+
+@SuppressLint("ClickableViewAccessibility")
+class DrawButtonView @JvmOverloads constructor(
```

Comment:
`DrawButton`!? This draws buttons!?!?!?

Kdoc and fix name lol

Reply:
Renamed to LineToolButton and has a Kdoc now